src/content/documentation/develop/debugging.md

```diff
@@ -90,6 +92,41 @@ You can now drag the toolbox tab to a separate window, so you can place it along
 
 {% capture content %}
 
+## Viewing log output
+
+Your extension can generate log messages, e.g. with the [Console API](https://developer.mozilla.org/docs/Web/API/Console_API) or by triggering (uncaught) errors.
+
+Log messages can usually be viewed via the [Console](https://developer.mozilla.org/docs/Tools/Web_Console) of the context where the script runs.
+The [Developer tools toolbox](#developer-tools-toolbox) at `about:debugging` shows log messages from extension scripts. For specific cases, see:
+
+  - [Debugging background scripts](#debugging-background-scripts)
+  - [Debugging options pages](#debugging-options-pages)
+  - [Debugging popups](#debugging-popups)
+  - [Debugging sidebars](#debugging-sidebars)
+  - [Debugging devtools panels](#debugging-developer-tools-pages-and-panels).
+
+Log messages from content scripts can be viewed in the developer tools of the tab where the content script runs instead of `about:debugging`, see:
+
+- [Debugging content scripts](#debugging-content-scripts)
+
+Log messages not associated with an extension script, such as the `stderr` output of the [nativeMessaging API](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Native_messaging)
+can only be viewed via the Browser Console. This console contains messages from the whole browser, including your and other extensions.
+It occasionally contains more detailed information about errors reported to your extension. For more information, see:
```

Comment:
It may be worth a mention that enabling that "Show Content Messages" toggle in the browser console may show logs not visible otherwise (not the case for the NativeMessaging.jsm though, and so we should try to not make it sound like it is mandatory for that).

Reply:
This is documented at that page. I've kept this overview minimal to offer a good signal-to-noise ratio (I've added the index in the first place because it was not easy to find the relevant documentation unless you already knew what you were looking for).